scripts/setup-k8s-master.sh.in

```diff
@@ -76,15 +65,37 @@ bootstrapTokens:
   usages:
   - signing
   - authentication
-controllerManagerExtraArgs:
-  cloud-provider: "aws"
-  allocate-node-cidrs: "false"
+nodeRegistration:
+  name: "${HOSTNAME}"
+  kubeletExtraArgs:
+    cloud-provider: "aws"
+---
+apiVersion: kubeadm.k8s.io/v1beta1
+kind: ClusterConfiguration
+kubernetesVersion: "$(cat /etc/kubernetes_community_ami_version)"
+apiServer:
+  timeoutForControlPlane: 4m0s
+  certSANs:
+  - "${LB_IPV4}"
+  extraArgs:
+    cloud-provider: "aws"
+clusterName: kubernetes
+controlPlaneEndpoint: "${LB_DNS}:443"
+controllerManager:
+  extraArgs:
+    cloud-provider: "aws"
+    allocate-node-cidrs: "false"
+etcd:
+  local:
+    dataDir: /var/lib/etcd
```

Comment:
this etcd section can go away as this is the default value

Reply:
👍 I missed that one. I'll wait for a little bit to see if I get a few other nits from other reviewers and batch the changes.